values/jobs/keycloak.gotmpl

```diff
@@ -23,12 +24,14 @@ tasks:
         repository: badouralix/curl-http2
       script: |
         {{ if $skipVerify }}export INSECURE='--insecure'{{ end }}
-        echo "Waiting until keycloak is accessible at https://keycloak.{{ $v.cluster.domain }}"
-        until $(curl $INSECURE --output /dev/null --silent --head --fail -I https://keycloak.{{ $v.cluster.domain }}); do 
+        echo "Waiting until keycloak is accessible at http://keycloak-http.keycloak.svc.cluster.local"
+        until $(curl $INSECURE --output /dev/null --silent --head --fail -I http://keycloak-http.keycloak.svc.cluster.local); do 
```

Comment:
don't do this as this is NOT the condition we are waiting for, as the keycloak task queries the jwks uri with public fqdn

Reply:
ok